client/scripts/legic.lua

```diff
@@ -217,8 +217,8 @@ end
 local currency = {
   ["03d2"]="EUR",
   ["0348"]="USD",
-  ["033A"]="GBP",
-  ["02F4"]="CHF"
+  ["033a"]="GBP",
+  ["02f4"]="CHF"
 }
```

Comment:
In fact I lowercase all those. The read function prints those as %x thus lowercase. It was mixed previously.